src/fu-self-test.c

```diff
@@ -4782,9 +4782,8 @@ fu_plugin_composite_release_sort_cb(gconstpointer a, gconstpointer b)
 }
 
 static void
-fu_plugin_composite_func(gconstpointer user_data)
+fu_plugin_composite_update_process(FuTest *self, guint scenario)
```

Comment:
I think this would be a lot clearer to duplicate the func, rather than conditionalizing on scenario

Reply:
separated.